src/NewsFeed/NewsFeed.tsx

```diff
@@ -5,11 +5,15 @@ import NewsCard from "../NewsCard/NewsCard";
 import { NewsFeedStory } from "../types";
 
 function NewsFeed() {
-  let [hasLoaded, setLoading] = useState(0);
-  let [randomStoriesData, setRandomStoriesData] = useState<string[]>([]);
-
-  let randomStoriesContent: NewsFeedStory[] = [];
-  let storyUrls: string[];
+  /*
+   * change loading message depending on value
+   * 0 - Loading...
+   * 1 - Almost there
+   * 2 - show content
+   */
+  let [isLoading, setLoading] = useState(2);
+  let [randomStoriesData] = useState<string[]>([]);
+  let [randomStoriesContent] = useState<NewsFeedStory[]>([]);
```

Comment:
Missing `set` functions for those two states.

Reply:
As they are arrays, I simply update them with `push`. I get "_____  is declared but its value is never read" warning if I would add the functions.